public/app/features/dashboard-scene/scene/DashboardControls.tsx

```diff
@@ -153,6 +153,22 @@ function DashboardControlsRenderer({ model }: SceneComponentProps<DashboardContr
 
   if (!model.hasControls()) {
     // To still have spacing when no controls are rendered
+
+    // Render variables that should be hidden but still require rendering. This is needed for Scopes, in order to get the react context.
+    const renderAsHiddenVariables = sceneGraph
+      .getVariables(dashboard)
+      .useState()
+      .variables.filter((v) => v.UNSAFE_renderAsHidden);
```

Comment:
Is this an okay approach, or is this a case where we should just look for `scopes`?